core/modules/loader/CentralClient.h

```diff
@@ -68,27 +69,20 @@ class KeyInfoData : public util::Tracker {
 class CentralClient : public Central {
 public:
     /// The client needs to know the master's IP and its own IP.
-    /// TODO The worker IP is temporary as it should be able to get
-    ///      that information from the master in the future. DM-16555
     CentralClient(boost::asio::io_service& ioService_,
-                  std::string const& masterHostName, int masterPort,
-                  int threadPoolSize, int loopSleepTime,
-                  std::string const& workerHostName, int workerPort,
-                  std::string const& hostName, int port)
-        : Central(ioService_, masterHostName, masterPort, threadPoolSize, loopSleepTime),
-          _workerHostName(workerHostName), _workerPort(workerPort),
-          _hostName(hostName), _udpPort(port) {
-        _server = std::make_shared<ClientServer>(ioService, _hostName, _udpPort, this);
-    }
+                  std::string const& hostName, ClientConfig const& cfg);
+
+    void start();
+
 
     ~CentralClient() override = default;
 
     std::string const& getHostName() const { return _hostName; }
     int getUdpPort() const { return _udpPort; }
     int getTcpPort() const { return 0; } ///< No tcp port at this time.
 
-    std::string getWorkerHostName() const { return _workerHostName; }
-    int getWorkerPort() const { return _workerPort; }
+    std::string getDefWorkerHost() const { return _defWorkerHost; }
+    int getDefWorkerPortUdp() const { return _defWorkerPortUdp; }
```

Comment:
It wasn't obvious to me that 'def' was 'default', but I think that's the case because now I see the comment near the member var. I'd probably prefer to not abbreviate Default, but if you want to abbreviate it I think the methods needed to be commented.